application/forms/Command/Object/ScheduleServiceDowntimeForm.php

```diff
@@ -192,19 +204,25 @@ protected function assembleElements()
                 'number',
                 'minutes',
                 [
-                    'required'  => true,
-                    'value'     => $this->flexibleDuration->m,
-                    'min'       => 0
+                    'required' => true,
+                    'value'    => $this->flexibleDuration->m,
+                    'min'      => 0
```

Comment:
Not sure what you're doing here, the indentation was previously (without your changes) already correct.